frontends/web/test/i18n/config.test.tsx

```diff
@@ -76,7 +76,7 @@ describe('language detector', () => {
     it('uses native-locale if userLanguage is null', done => {
```

Comment:
@x1ddos change to `'uses native-locale if userLanguage is an empty string'` ?

Reply:
done